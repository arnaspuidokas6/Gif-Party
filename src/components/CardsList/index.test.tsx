import { render, screen } from "@testing-library/react";
import { CardsList } from ".";
describe('Cards List', () => {
    it('should show no gifs', () => {
        const { container } = render(<CardsList />);

        expect(screen.getByTestId('no-gifs')).toBeVisible();
        expect(container).toMatchSnapshot();
    });
});
