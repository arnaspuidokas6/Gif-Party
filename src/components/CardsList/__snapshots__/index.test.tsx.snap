// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards List should show no gifs 1`] = `
<div>
  <h1
    class="mt-20 font-bold text-3xl text-gray-900"
    data-testid="no-gifs"
  >
    Sorry, no gifs :( try typing in search.
  </h1>
</div>
`;
